server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -436,4 +410,9 @@ public GcCycleMetrics getGcCycleMetrics() {
     return gcCycleMetrics;
   }
 
+  @VisibleForTesting
+  public FileJanitor getFileJanitor() {
+    return this.janitor;
+  }
+
```

Comment:
This can be package-private. It doesn't need to be public for unit tests in the same package.